__tests__/pet.test.js

```diff
@@ -0,0 +1,112 @@
+const Pet = require("../src/pet");
+
+describe("constructor", () => {
+  const pet = new Pet("Fido");
+  it("returns an object", () => {
+    expect(new Pet()).toBeInstanceOf(Object);
+  });
+  it("renames pet", () => {
+    expect(pet.name).toBe("Fido");
+  });
+  it("has an initial age of 0", () => {
```

Comment:
For these values, would it be better to define them in some variables at the top so they can be referenced throughout the file in the same way as in `pet.js`?